xbmc/addons/Visualization.cpp

```diff
@@ -143,4 +153,16 @@ void CVisualization::transfer_preset(void* kodiInstance, const char* preset)
   addon->m_presets.emplace_back(preset);
 }
 
+void CVisualization::clear_presets(void* kodiInstance)
+{
+  CVisualization* addon = static_cast<CVisualization*>(kodiInstance);
```

Comment:
pls use reinterpret_cast for void pointers

Reply:
About this I'm unsure and better to leave on `static_cast`, seen always that `reinterpet`should only be used if nothing else works.

Know the following:
https://stackoverflow.com/questions/332030/when-should-static-cast-dynamic-cast-const-cast-and-reinterpret-cast-be-used
https://stackoverflow.com/questions/573294/when-to-use-reinterpret-cast
https://stackoverflow.com/questions/4476182/which-cast-to-use-static-cast-or-reinterpret-cast

Reply:
You're right! You should leave it as static_cast. I was wrong, I thought it is good practice to use reinterpret_cast for void pointer casts.